bigchaindb/web/views/parameters.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def valid_txid(txid):
+    if re.match('^[a-fA-F0-9]{64}$', txid):
+        return txid.lower()
+    raise ValueError("Invalid hash")
+
+
+def valid_bool(val):
+    if val == 'true':
+        return True
+    if val == 'false':
+        return False
+    raise ValueError('Boolean value must be "true" or "false" (lowercase)')
```

Comment:
I decided to leave the "boolean" and "operation" argument parsers, even though they could be supported by `choice` parameter in `parser.add_argument`, because having them separate promotes re-use. Put another way, if I wanted to have an "operation" parameter in another http endpoint, using `parser.add_argument` I'd have to re-formulate it, wheras having the `valid_operation` function separate it easily encapsulates the valid / invalid logic and the error message.